src/main.js

```diff
@@ -4,6 +4,7 @@ import store from './store';
 import App from './App.vue';
 import { VTooltip, VPopover, VClosePopover } from 'v-tooltip';
 import axios from 'axios';
+import filter from './filters/filter'
```

Comment:
According to [this link](https://stackoverflow.com/a/45054257/9614982), you don't need to specify the filters into Vue instance above, just importing it into this file, do you?